sql-plugin/src/main/scala/com/nvidia/spark/rapids/HostColumnarToGpu.scala

```diff
@@ -407,6 +291,7 @@ case class HostColumnarToGpu(child: SparkPlan, goal: CoalesceSizeGoal)
     OP_TIME -> createNanoTimingMetric(MODERATE_LEVEL, DESCRIPTION_OP_TIME),
     STREAM_TIME -> createNanoTimingMetric(MODERATE_LEVEL, DESCRIPTION_STREAM_TIME),
     CONCAT_TIME -> createNanoTimingMetric(MODERATE_LEVEL, DESCRIPTION_CONCAT_TIME),
+    BUFFER_TIME -> createNanoTimingMetric(MODERATE_LEVEL, DESCRIPTION_BUFFER_TIME),
```

Comment:
If we are going to use buffer time here we are going to need to update the docs https://github.com/NVIDIA/spark-rapids/blob/branch-22.04/docs/tuning-guide.md#metrics. I think it fits, but the docs make it specific for buffering input data. There is also a tool that will use bufferTime to graph how long each process took to buffer input data. Because of both of these things I think it might be better to add in a new metric. But even then we would need to update the docs to explain that new metric.

I also would prefer to see this at a DEBUG_LEVEL. I don't think we need this all the time.

Reply:
Done.